### PR TITLE
page format: lowercase command names consistently

### PR DESCRIPTION
**Pull request by waldyrious**
Inspired by @liuderchi's #1081 and others before that: #967, #965, #947, #699, #658, #534, #313, #312, #309, and #308 (Phew! I think I got them all. Maybe we should create a new label for large-scale cleanups like these).

### CODE REVIEW

File: pages/common/parallel.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# GNU Parallel
+# parallel
 
-> Run commands on multiple CPU cores.
+> GNU Parallel: Run commands on multiple CPU cores.
```
Comment (agnivade) [inline pages/common/parallel.md]: Just wondering what is the benefit of appending "GNU Parallel:" ?

Comment (waldyrious) [inline pages/common/parallel.md]: Good point. I'll remove it.

**Comment by agnivade** [comment]
:+1:

**Comment by waldyrious** [comment]
Let's to this! LGTM too :P

**Pull request merged**